tests/query/snql/test_query.py

```diff
@@ -143,12 +157,72 @@
         ),
         id="Basic query with no spaces and no ambiguous clause content",
     ),
+    pytest.param(
+        "MATCH (e: events) SELECT count() AS count BY tags[key], measurements[lcp.elementSize] WHERE a<3 AND measurements[lcp.elementSize] > 1",
+        LogicalQuery(
+            {},
+            QueryEntity(
+                EntityKey.EVENTS, get_entity(EntityKey.EVENTS).get_data_model(), "e",
+            ),
+            selected_columns=[
+                SelectedExpression(
+                    "count", FunctionCall("_snuba_count", "count", tuple()),
+                ),
+                SelectedExpression(
+                    "tags[key]",
+                    SubscriptableReference(
+                        "_snuba_tags[key]",
+                        Column("_snuba_tags", None, "tags"),
+                        Literal(None, "key"),
+                    ),
+                ),
+                SelectedExpression(
+                    "measurements[lcp.elementSize]",
+                    SubscriptableReference(
+                        "_snuba_measurements[lcp.elementSize]",
+                        Column("_snuba_measurements", None, "measurements",),
+                        Literal(None, "lcp.elementSize"),
+                    ),
+                ),
+            ],
+            groupby=[
+                SubscriptableReference(
+                    "_snuba_tags[key]",
+                    Column("_snuba_tags", None, "tags"),
+                    Literal(None, "key"),
+                ),
+                SubscriptableReference(
+                    "_snuba_measurements[lcp.elementSize]",
+                    Column("_snuba_measurements", None, "measurements",),
+                    Literal(None, "lcp.elementSize"),
+                ),
+            ],
+            condition=binary_condition(
+                None,
```

Comment:
I merged yesterday a change that removes aliases from binary_condition. This argument should be removed.